.github/workflows/bcny-ci.yml

```diff
@@ -108,3 +108,11 @@ jobs:
           personalAccessToken: ${{ secrets.SYMBOL_SERVER_PAT }}
           symbolsFolder: ${{ github.workspace }}\sentry-native-development\usr\bin
           searchPattern: '**/*.pdb'
+
+      - name: Upload DLLs to Azure
+        uses: microsoft/action-publish-symbols@719c40b80e38bca806f3e01e1e3dd3a67554cd68
+        with:
+          accountName: thebrowsercompany
+          personalAccessToken: ${{ secrets.SYMBOL_SERVER_PAT }}
+          symbolsFolder: ${{ github.workspace }}\sentry-native-development\usr\bin
+          searchPattern: '**/*.dll'
```

Comment:
Can we not merge this into the previous step?

Reply:
From what I read online this action doesn't work well when using multiple patterns, so I used the simple approach of duplicating the step here. We could try though! What would be the syntax to match multiple extensions you think?

Reply:
It should work fine I believe it just uses the `glob` module. You can do `**/*.{dll,exe,pdb}` or `**/*@(dll|exe|pdb)`

Reply:
Testing that now!

Reply:
The `**/*.@(dll|exe|pdb)` syntax doesn't work: https://github.com/thebrowsercompany/sentry-native/actions/runs/8255409188/job/22581789250?pr=8#step:11:59 , testing the other one now...

Reply:
Other pattern also doesn't work: https://github.com/thebrowsercompany/sentry-native/actions/runs/8255475468/job/22582010400?pr=8#step:11:53